Solutions/Armorblox/Data Connectors/ArmorbloxAzureSentinelConnector/__init__.py

```diff
@@ -35,12 +36,13 @@
     raise Exception("Armorblox Data Connector: Invalid Log Analytics URI")
 
 if (ARMORBLOX_INSTANCE_NAME == "") and (ARMORBLOX_INSTANCE_URL == ""):
-   raise Exception("At least one of Armorblox instance name or URL need to be provided")
+    raise Exception("At least one of Armorblox instance name or URL need to be provided")
```

Comment:
What is this whitespace change for?

Reply:
It is an indentation error. There was space missing as it was not a tab. So, we added a space over there.